ArcGISRuntimeSDKQt_CppSamples/Analysis/FindServiceAreasForMultipleFacilities/FindServiceAreasForMultipleFacilities.cpp

```diff
@@ -0,0 +1,178 @@
+// [WriteFile Name=FindServiceAreasForMultipleFacilities, Category=Analysis]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+#include "FindServiceAreasForMultipleFacilities.h"
+
+#include "FeatureLayer.h"
+#include "GraphicsOverlay.h"
+#include "Map.h"
+#include "MapQuickView.h"
+#include "PictureMarkerSymbol.h"
+#include "QueryParameters.h"
+#include "ServiceAreaParameters.h"
+#include "ServiceAreaPolygon.h"
+#include "ServiceAreaResult.h"
+#include "ServiceAreaTask.h"
+#include "ServiceFeatureTable.h"
+#include "SimpleFillSymbol.h"
+#include "SimpleRenderer.h"
+#include "TaskWatcher.h"
+
+#include <QUrl>
+
+using namespace Esri::ArcGISRuntime;
+
+namespace
+{
+const QUrl url("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Facilities/FeatureServer/0");
+const QUrl imageUrl("https://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png");
```

Comment:
Can/should this be local data? I've never seen images utilized from a URL before.

Reply:
The other platforms used a URL. I could download the png file but I thought it would be better to follow the other samples? @anmacdonald

Reply:
Very strange! @ldanzinger ?

Reply:
Yeah this works. We support that